presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorMetadata.java

```diff
@@ -231,8 +231,9 @@ default TableStatistics getTableStatistics(ConnectorSession session, ConnectorTa
 
     /**
      * Creates a schema.
+     * @param principal TODO
```

Comment:
remove

Reply:
Apologies, Eclipse generated that and I missed it.